sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -266,7 +266,7 @@ static synchronized void unsetInstance() {
      * @param throwable The throwable describing the handled error.
      */
     @SuppressWarnings({"SameParameterValue", "WeakerAccess"})
-    public static void trackError(Throwable throwable) {
+    public static void trackError(@NonNull Throwable throwable) {
```

Comment:
If I understand it correctly, it doesn't check for `null`, but generates a warning if I pass a `null` here.
I think we should check with `if` as well, shouldn't we?

Reply:
Decided to avoid adding additional conditions. Annotation should be enough.